Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles.csproj

```diff
@@ -8,7 +8,7 @@
 	  <AssemblyName>Frends.SFTP.DownloadFiles</AssemblyName>
 	  <RootNamespace>Frends.SFTP.DownloadFiles</RootNamespace>
 
-	  <Version>2.5.4</Version>
+	  <Version>2.5.5</Version>
```

Comment:
Since there's a modification to the result object, should there be an update to the minor version instead of the patch? (2.6.0)